google/ggrc-core
# GGRC-6542 During cycle set up, task dates that fall on US public holidays are forcibly moved to a different day

# Issue description

**USER PROBLEM:**

What is the problem / issue?
When setting up a cycle, if a task date falls on a US public holiday gGRC forces the date to be a different date in the "Next Cycle".
E.g. 11 Nov is a public holiday and gets moved to the 8 Nov 
Eg. 14 Oct is a public holiday and gets moved to 11 Oct 

**What is the expected result / outcome?**
Dates should not be automatically changed if they fall on a US public holiday.

# Steps to test the changes

Set up a cycle and put dates that are public US holidays. Check this dates haven't been changed.

# Solution description

I've just deleted this holidays logic. At last.

# Sanity checklist




# PR Review checklist
